src/pages/get-instance.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { SEO } from 'components';
+import { Helmet } from 'react-helmet';
+
+const TYPEFORM_ID = 'tR5Cpfis';
+const BASE_FORM_URL = 'https://form.typeform.com/to';
+
+const GetInstanceRequest = ({ data, location }) => {
+  const siteTitle = data.site.siteMetadata.title;
+
+  const params = new URLSearchParams(location.search.substring(1));
+  const referredEmail = params.get('referred_email');
+  const clickedButtonLabel = params.get('clicked_button_label');
+  const hashParams = `referred_email=${referredEmail}&clicked_button_label=${clickedButtonLabel}`;
+
+  return (
+    <>
```

Comment:
Whats this do?

Reply:
Which bit? The `<>`?

It groups react elements without adding an actual HTML tag. It's called a fragment. https://reactjs.org/docs/fragments.html